<ui version="4.0" >
 <class>RavrProgQt</class>
 <widget class="QDialog" name="RavrProgQt" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>560</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>519</width>
    <height>560</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>519</width>
    <height>560</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>RavrProgQt</string>
  </property>
  <widget class="QTextEdit" name="textEdit" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>440</y>
     <width>501</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <antialiasing>true</antialiasing>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="undoRedoEnabled" >
    <bool>false</bool>
   </property>
   <property name="readOnly" >
    <bool>true</bool>
   </property>
   <property name="acceptRichText" >
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>501</width>
     <height>421</height>
    </rect>
   </property>
   <property name="currentIndex" >
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_prog" >
    <attribute name="title" >
     <string>Programmer</string>
    </attribute>
    <widget class="QWidget" name="verticalLayout_5" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>10</y>
       <width>471</width>
       <height>371</height>
      </rect>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="1" column="1" colspan="2" >
       <widget class="QComboBox" name="InterfaceCombo" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>7</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="10" column="1" colspan="2" >
       <widget class="QLineEdit" name="SigEdit" >
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="label_14" >
        <property name="text" >
         <string>Clock Generator:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>Programming Clock:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_18" >
        <property name="text" >
         <string>Select Interface:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="3" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1" colspan="2" >
       <widget class="QComboBox" name="ProgrammerCombo" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>7</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maxVisibleItems" >
         <number>15</number>
        </property>
       </widget>
      </item>
      <item row="9" column="1" colspan="2" >
       <widget class="QComboBox" name="DeviceCombo" >
        <property name="maxVisibleItems" >
         <number>15</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="label_16" >
        <property name="text" >
         <string>Select Programmer:</string>
        </property>
       </widget>
      </item>
      <item row="11" column="1" colspan="2" >
       <widget class="QLineEdit" name="SerialEdit" >
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="3" >
       <widget class="QPushButton" name="pushButton_Connect" >
        <property name="text" >
         <string>&amp;Connect</string>
        </property>
       </widget>
      </item>
      <item row="11" column="0" >
       <widget class="QLabel" name="label_8" >
        <property name="text" >
         <string>AVR Serial Number:</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Signature + JTAG ID:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2" >
       <widget class="QPushButton" name="pushButton_Refresh" >
        <property name="maximumSize" >
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text" >
         <string>&amp;Refresh</string>
        </property>
       </widget>
      </item>
      <item row="12" column="1" >
       <widget class="QComboBox" name="OsccalCombo" >
        <property name="maxVisibleItems" >
         <number>8</number>
        </property>
       </widget>
      </item>
      <item row="12" column="0" >
       <widget class="QLabel" name="label_7" >
        <property name="text" >
         <string>Osccal Values:</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0" >
       <widget class="QLabel" name="label_4" >
        <property name="text" >
         <string>Select Device:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" >
       <widget class="QLabel" name="label_5" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>1</hsizetype>
          <vsizetype>5</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Select Port:</string>
        </property>
       </widget>
      </item>
      <item row="12" column="2" >
       <widget class="QPushButton" name="pushButton_Refresh_AVR" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="maximumSize" >
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text" >
         <string>&amp;Refresh</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="3" >
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>Device Information</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="1" >
       <widget class="QComboBox" name="PortCombo" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
        <property name="minimumSize" >
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="editable" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="8" column="0" colspan="3" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="1" >
       <widget class="QComboBox" name="PClockCombo" >
        <property name="editable" >
         <bool>false</bool>
        </property>
        <property name="maxVisibleItems" >
         <number>20</number>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="QComboBox" name="EClockCombo" >
        <property name="editable" >
         <bool>false</bool>
        </property>
        <property name="maxVisibleItems" >
         <number>22</number>
        </property>
       </widget>
      </item>
      <item row="2" column="2" >
       <widget class="QPushButton" name="pushButton_ApplyP" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>&amp;Apply</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2" >
       <widget class="QPushButton" name="pushButton_ApplyE" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>A&amp;pply</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_flash" >
    <attribute name="title" >
     <string>Flash</string>
    </attribute>
    <widget class="QWidget" name="verticalLayout_2" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>10</y>
       <width>471</width>
       <height>371</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>20</width>
          <height>16</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_ChipErase" >
        <property name="text" >
         <string>&amp;Chip Erase</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>20</width>
          <height>16</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_Erase" >
          <property name="text" >
           <string>Erase &amp;before writing flash</string>
          </property>
          <property name="checked" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_Verify" >
          <property name="text" >
           <string>Verify &amp;after writing</string>
          </property>
          <property name="checked" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>20</width>
          <height>16</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2" >
        <property name="title" >
         <string>Flash</string>
        </property>
        <widget class="QWidget" name="verticalLayout_4" >
         <property name="geometry" >
          <rect>
           <x>9</x>
           <y>19</y>
           <width>451</width>
           <height>92</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QLineEdit" name="lineEdit_Flash" />
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <widget class="QPushButton" name="pushButton_Read" >
              <property name="text" >
               <string>&amp;Read</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_Write" >
              <property name="text" >
               <string>&amp;Write</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_Verify" >
              <property name="text" >
               <string>&amp;Verify</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_Select" >
              <property name="text" >
               <string>&amp;Select File</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox" >
        <property name="title" >
         <string>Eeprom</string>
        </property>
        <widget class="QWidget" name="verticalLayout_6" >
         <property name="geometry" >
          <rect>
           <x>10</x>
           <y>20</y>
           <width>451</width>
           <height>92</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QLineEdit" name="lineEdit_Eeprom" />
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <layout class="QHBoxLayout" >
              <property name="margin" >
               <number>0</number>
              </property>
              <property name="spacing" >
               <number>6</number>
              </property>
              <item>
               <widget class="QPushButton" name="pushButton_Read_2" >
                <property name="text" >
                 <string>Rea&amp;d</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_Write_2" >
                <property name="text" >
                 <string>Writ&amp;e</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_Verify_2" >
                <property name="text" >
                 <string>Verif&amp;y</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_Select_2" >
                <property name="text" >
                 <string>Selec&amp;t File</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar" >
        <property name="value" >
         <number>0</number>
        </property>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_fuse" >
    <attribute name="title" >
     <string>Fuses</string>
    </attribute>
    <widget class="QWidget" name="horizontalLayout_7" >
     <property name="geometry" >
      <rect>
       <x>9</x>
       <y>319</y>
       <width>471</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QLabel" name="label_10" >
        <property name="text" >
         <string>Fuses:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_Fuses" >
        <property name="readOnly" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayout_5" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>350</y>
       <width>471</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QPushButton" name="pushButton_ReadFuse" >
        <property name="text" >
         <string>&amp;Read</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_WriteFuse" >
        <property name="text" >
         <string>&amp;Write</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_VerifyFuse" >
        <property name="text" >
         <string>&amp;Verify</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_DefaultFuse" >
        <property name="text" >
         <string>&amp;Default</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
